packages/@ember/-internals/owner/index.ts

```diff
@@ -222,37 +305,162 @@ export function setOwner(object: object, owner: Owner): void {
   glimmerSetOwner(object, owner);
 }
 
-export interface ContainerMixin extends Owner {
-  ownerInjection(): void;
+// Defines the type for the ContainerProxyMixin. When we rationalize our Owner
+// *not* to work via mixins, we will be able to delete this entirely: this
+// overload for `lookup()` and all of `ownerInjection()` will go away.
```

Comment:
I don't see an overload for `lookup()` here—did that change after this comment was written?

Reply:
It did, and I thought I had deleted it; thanks for the catch here as well!